### PR TITLE
adds skeleton shop page

### PR DESCRIPTION
**Pull request by dandelionadia**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,29 +1,29 @@
...
-import { Shop } from './pages/Shop'
-import { Header } from './components/Header'
-import { BrowserRouter as Router, Switch, Route } from "react-router-dom"
+import React from "react";
+import { HomePage } from "./pages/HomePage";
+import "./style/main.scss";
+import Layout, { Box } from "atomic-layout";
+import { About } from "./pages/About";
+import { Shop } from "./pages/Shop";
+import { Header } from "./components/Header";
```
Comment (kettanaito) [inline src/App.js]: I suggest you press `CTRL + S` in this file to reformat it.

File: src/atoms/Burger.js [hunk 2]
```diff
@@ -1,30 +1,30 @@
...
-            <div onClick={onClick}>
-                <StyleBtnLine></StyleBtnLine>
-                <StyleBtnLineBlue></StyleBtnLineBlue>
-                <StyleBtnLine></StyleBtnLine>
-            </div>
-        </Box>
-    );
+  return (
+    <Box flex flexDirection="column" justifyContent="center" height="100%">
+      <div onClick={onClick}>
```
Comment (kettanaito) [inline src/atoms/Burger.js]: Maybe make it a button? `div` is not a clickable element.

File: src/components/Pictures.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  }
+`
+
+function Pictures({ items }) {
+  return (
+    <div>
+      <Composition templateCols="repeat(3, 1fr)" templateColsLg="1fr">
+        {items.map(item => (
+          <Box as={StyledImage} padding="1rem" margin="1rem">
+            <img className="background__img" src={item.img} />
```
Comment (kettanaito) [inline src/components/Pictures.js]: Do you use this class name in CSS?

File: src/components/Header.js [hunk 4]
```diff
@@ -1,11 +1,10 @@
-import React from 'react';
-import { Composition, Only } from 'atomic-layout';
+import React from 'react'
+import { Composition, Only } from 'atomic-layout'
```
Comment (kettanaito) [inline src/components/Header.js]: `Only` is no longer used. Remove it.

File: src/pages/Shop.js [hunk 5]
```diff
@@ -1,21 +1,84 @@
...
+  contentRight
+`
+const areasLarge = `
+  contentLeft image contentRight
+  / 1fr 60% 1fr
+`
+
+const pictures = [
+  {
+    img:
```
Comment (kettanaito) [inline src/pages/Shop.js]: Consider using `src:` instead of `img:`.

File: src/pages/Shop.js [hunk 6]
```diff
@@ -1,21 +1,84 @@
...
+    <div>
+      <Composition
+        areas={areasMobile}
+        templateLg={areasLarge}
+        padding={1}
+        gap={2}
+      >
+        {Areas => (
+          <>
             <GlobalStyle bgColor="#FCF2E5" />
```
Comment (kettanaito) [inline src/pages/Shop.js]: I recommend you to render this as close to the root of this component as possible.
Maybe as the first child of `div`:

```jsx
<div>
  <GlobalStyle {...} />
```

File: src/components/Pictures.js (hunk 3 above)
Comment (dandelionadia) [reply src/components/Pictures.js]: Yes!
I added it for all image.

File: src/components/Header.js [hunk 7]
```diff
@@ -16,48 +15,48 @@ const areasTablet = `
...
-                )}
-            </Composition>
-        )
-    }
+  state = {
+    isMenuOpen: false
+  }
+
+  handleMenuClick = () => {
+    console.log('okay')
```
Comment (kettanaito) [inline src/components/Header.js]: You can remove this console statement.

File: src/components/MenuMob.js [hunk 8]
```diff
@@ -1,42 +1,44 @@
...
-    position: relative;
-    list-style: none;
+  padding: 1rem;
+  text-transform: uppercase;
+  font-weight: 600;
+  position: relative;
+  list-style: none;
 `
 
 function MenuMob({ isOpen }) {
```
Comment (kettanaito) [inline src/components/MenuMob.js]: `MenuMobile`?

File: src/components/Pictures.js [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+
+  :hover {
+    transition: background-color 1.5s;
+    background-color: black;
+  }
+`
+
+function Pictures({ items }) {
+  return (
+    <div>
```
Comment (kettanaito) [inline src/components/Pictures.js]: There is no need for an extra `div`. 

`Composition` itself is a `div` by default. Please remove the parent div.

File: src/pages/HomePage.js [hunk 10]
```diff
@@ -19,25 +19,34 @@ const areasTablet = `
...
     <Composition
-        areas={areasMobile}
-        areasMd={areasTablet}
-        padding={2}
-        templateColsMd="40% auto"
-        gap={2}
+      areas={areasMobile}
+      areasMd={areasTablet}
+      padding={2}
+      templateColsMd="40% auto"
```
Comment (kettanaito) [inline src/pages/HomePage.js]: You can also combine `areas` and `templateCols` using the `template` prop:

```jsx
const areasTablet = `
  info image
  / 40% auto
`

<Composition
  areas={areasMobile}
  templateMd={areasTablet} // note the prop "template"
>
```

File: src/pages/Shop.js [hunk 11]
```diff
@@ -1,21 +1,84 @@
...
+    src:
+      'https://cdn.dribbble.com/users/1355613/screenshots/6248741/mr._duck_in_the_rain.jpg'
+  },
+  {
+    src:
+      'https://cdn.dribbble.com/users/1355613/screenshots/6284276/attachments/1346008/customer_tags_2.png'
+  }
+]
+
+const index = pictures.length / 2
```
Comment (kettanaito) [inline src/pages/Shop.js]: Please don't do this logic outside of the `Shop` function.

Move these three `const`'s into the body of the `Shop` function.

Comment (kettanaito) [inline src/pages/Shop.js]: Try to have an odd number of `pictures` (for example 7 pictures).
Then this division by 2 will not work properly.

You need to _round_ the number to the closest smallest or highest:

```js
// pictures.length is 7
const index = pictures.length / 2 // 3.5 - that's not okay, need a whole number
const index = Math.ceil(pictures.length / 2) // 4 - rouding to the highest: 3.5 -> 4
const index = Math.floor(pictures.length / 2) // 3 - rounding to the lowest: 3.5 - > 3
```

File: src/pages/HomePage.js (hunk 10 above)
Comment (dandelionadia) [reply src/pages/HomePage.js]: I can't
it breaks this part

**Pull request merged**